components/prism-yaml.js

```diff
@@ -0,0 +1,44 @@
+Prism.languages.yaml = {
+	'scalar': {
+		pattern: /([\-:]\s*(![^\s]+)?[ \t]*[\|>])[ \t]*(?:(\n[ \t]+)[^\r\n]+)?(?:\3[^\r\n]+)*/,
```

Comment:
If I read this correctly, the `\3` refers to the group `(\n[ \t]+)` which is part of an optional group. Shoudn't the last bit `(?:\3[^\r\n]+)*` be included in the same optional group (as the `\3` is mandatory)?